package.json

```diff
@@ -34,6 +34,7 @@
     "helmet": "^3.21.2",
     "jsonschema": "<1.3.0",
     "kleur": "^4.1.1",
+    "mongoose-aggregate-paginate-v2": "^1.0.42",
```

Comment:
I'm relatively new to Node and `npm`, so this isn't a change request as much as my putting a pin in something that sticks out to me. This package has a number of qualities that, without further inspection, make me worry about its use in a production application. Those are:
- It contains a version `v2` in its name
- The bug fix version number is uncharacteristically high versus the major and minor version numbers
- It hasn't been updated in seven months
- It only has five total contributors
- I can't find a change log anywhere

How are we vetting packages we add to the project? Is this package state common for Node applications? To make a manufacturing analogy, I want to make sure we're sourcing reliable, quality-tested parts instead of an untested third-party replica with no assurance that it won't break after six months of use.

@mattrbianchi might have some insight here.